tests/unit/remote/gdrive/conftest.py

```diff
@@ -0,0 +1,15 @@
+import pytest
+
+from dvc.repo import Repo
+from dvc.remote.gdrive import RemoteGDrive
+
+
+@pytest.fixture()
+def repo():
+    return Repo(".")
+
+
+@pytest.fixture
+def gdrive(repo):
+    ret = RemoteGDrive(repo, {"url": "gdrive://root/data"})
```

Comment:
Seems like we could totally just use None
```suggestion
    ret = RemoteGDrive(None, {"url": "gdrive://root/data"})
```